homeassistant/components/http.py

```diff
@@ -45,6 +51,63 @@
 }, extra=vol.ALLOW_EXTRA)
 
 
+def collect_hash(config):
+    """
+    find the password/hash information in http configuration.
+
+    Args:
+        config: configuration['http']
+
+    Returns:
+        '$<algorithm>$<salt>$<hash>'
+        or
+        None if password or hash are not configured
+    """
+    if CONF_API_HASH in config:
+        hash_ = config[CONF_API_HASH]
```

Comment:
It might be better form to name this variable something like api_hash. When reading code, hash_ looks a lot like hash.